bpf/kmesh/sockops_workload.c

```diff
@@ -0,0 +1,70 @@
+/*
+ * Copyright 2023 The Kmesh Authors.
```

Comment:
2024
Please check all the files for similar errors.

Reply:
done